src/Repository/WorklogRepository.php

```diff
@@ -192,7 +191,10 @@ public function findBillableWorklogsByWorkerAndDateRange(\DateTime $dateFrom, \D
             ));
 
         if (null !== $isBilled) {
-            $qb->andWhere($qb->expr()->eq('worklog.isBilled', ':isBilled'));
+            $qb->andWhere($qb->expr()->orX(
```

Comment:
This does not seem like the correct fix to the problem.
With this code, if :isBilled is true, it will include the result if worklog.isBilled is null...

If there is a problem with the data from leantime data provider, it should be fixed in the synchronization instead.